src/pybind/mgr/requirements.txt

```diff
@@ -0,0 +1,8 @@
+attrs==17.4.0
+coverage==4.5.2
+pytest==3.3.2
+pluggy==0.6.0
+pytest-cov==2.5.1
+py==1.5.2
```

Comment:
All the packages are installed because of `pytest-cov`, I will reduce the list to just `pytest-cov` than pip will install all the following packages automatically.

For python 2 -> `py2 installed: atomicwrites==1.3.0,attrs==19.1.0,configparser==4.0.2,contextlib2==0.5.5,coverage==4.5.4,funcsigs==1.0.2,importlib-metadata==0.22,more-itertools==5.0.0,packaging==19.1,pathlib2==2.3.4,pluggy==0.13.0,py==1.8.0,pyparsing==2.4.2,pytest==4.6.5,pytest-cov==2.7.1,scandir==1.10.0,six==1.12.0,wcwidth==0.1.7,zipp==0.6.0
`

For python 3 -> `py3 installed: atomicwrites==1.3.0,attrs==19.1.0,coverage==4.5.4,importlib-metadata==0.22,more-itertools==7.2.0,packaging==19.1,pluggy==0.13.0,py==1.8.0,pyparsing==2.4.2,pytest==5.1.2,pytest-cov==2.7.1,six==1.12.0,wcwidth==0.1.7,zipp==0.6.0
`